tutorials/tutorials.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd $DIR
+$DIR/build.sh
+
+docker run -e APPLITOOLS_API_KEY tutorial_python_basic
+docker run -e APPLITOOLS_API_KEY tutorial_python_ufg
```

Comment:
Would you add applitools images tutorial?

Reply:
Sure, I could add repo: https://github.com/applitools/tutorial-images-python as test for the images. But on tutorials page it refers to the code snippet instead of repo (https://applitools.com/tutorials/screenshots-python.html).